features/features/package_managers/glide_spec.rb

```diff
@@ -7,22 +7,35 @@
 
   context 'when project is in src directory' do
     specify 'are shown in reports for a project' do
-      project = LicenseFinder::TestingDSL::GlideProject.create
-      ENV['GOPATH'] = "#{project.project_dir}/gopath_glide"
+      LicenseFinder::TestingDSL::GlideProject.create
 
-      go_developer.run_license_finder('gopath_glide')
+      go_developer.run_license_finder('src/gopath_glide/src')
       expect(go_developer).to be_seeing_line 'github.com/Masterminds/semver, 517734cc7d6470c0d07130e40fd40bdeb9bcd3fd, MIT'
       expect(go_developer).to be_seeing_line 'github.com/Masterminds/cookoo, 78aa11ce75e257c51be7ea945edb84cf19c4a6de, MIT'
     end
   end
 
-  context 'when projecy is not in src directory' do
+  context 'when project in root directory' do
     specify 'are shown in reports for a project' do
       LicenseFinder::TestingDSL::GlideProjectWithoutSrc.create
 
-      go_developer.run_license_finder('gopath_glide_without_src')
+      go_developer.run_license_finder('src/gopath_glide_without_src')
```

Comment:
Same question about the parent src directory

Reply:
All those changes is to reflect a proper go project setup.  GOPATH must have a `src` subdirectory or else glide will not work.  And all project usually are located inside the GOPATH/src directory.  So, these changes reflect a better GO environment setup for testing, and is closer to what NORSK does as well.